config/package-solution.json

```diff
@@ -32,6 +32,14 @@
       {
         "resource": "Microsoft Graph",
         "scope": "User.ReadBasic.All"
+      },
+      {
+        "resource": "Microsoft Graph",
+        "scope": "Group.Read.All"
+      },
+      {
+        "resource": "Microsoft Graph",
+        "scope": "Directory.Read.All"
```

Comment:
Why do we need directory.read.all? This overwrite group.read.all.
If we need to access more information from the user, is User.Read.All work?

Reply:
To properly check if the current user is a Sharepoint admin, we need this permission to return the piece of data that we check against. Without it the roleTemplateId value will return null and block the user.

From the small tests I did, User.Read.All wasn't returning all the values the extension needs.